Updated crash dump info in CONTRIBUTING.md (#10664)
microsoft/WSL: CONTRIBUTING.md

The easiest way to report a WSL process crash is by [collecting a user-mode crash dump](https://learn.microsoft.com/en-us/windows/win32/wer/collecting-user-mode-dumps).

> To force WSL to crash and collect this information, please open a PowerShell prompt with admin privileges, navigate to a folder where you'd like to put your log files and run these commands: 

Reviewer: This script won't technically force WSL to crash. Maybe we could rewrite as "To collect dumps of all running WSL processes" ?

> To force WSL to crash and collect this information, please open a PowerShell prompt with admin privileges, navigate to a folder where you'd like to put your log files and run these commands: 

Author: ```suggestion
To collect dumps of all running WSL processe, please open a PowerShell prompt with admin privileges, navigate to a folder where you'd like to put your log files and run these commands: 
```
